packages/odyssey-react-mui/src/themes/odyssey/shape.ts

```diff
@@ -15,4 +15,6 @@ import * as Tokens from "@okta/odyssey-design-tokens";
 
 export const shape: ThemeOptions["shape"] = {
   borderRadius: Tokens.BorderRadiusBase,
+  // Expected: Mods should work identically to mixins.ts/mixins.types.ts
+  borderStyle: Tokens.BorderStyleBase,
```

Comment:
Result:

```
Type '{ borderRadius: string; borderStyle: string; }' is not assignable to type 'Partial<Shape>'.
  Object literal may only specify known properties, and 'borderStyle' does not exist in type 'Partial<Shape>'.
```

Reply:
This is the same error that presents if `shape.types.ts` is empty.

Reply:
Weird. This is one of those situations where it's easier to play around with it trial 'n error.